### PR TITLE
add machine learning bucket

### PR DESCRIPTION
**Pull request by sophiesongge**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,8 @@ classes
 .gradle
 gradle/ext/
 /bin/
+.DS_Store
```
Comment (mbenguig) [inline .gitignore:11]: must be gitted?

File: .gitignore [hunk 2]
```diff
@@ -8,6 +8,8 @@ classes
...
+
```
Comment (mbenguig) [inline .gitignore:12]: must be gitted?

File: src/main/resources/default-workflows/Machine_Learning/DLM_AnoDet_a_NS.xml [hunk 3]
```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="DLM_AnoDet_a_NS" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2">
```
Comment (mbenguig) [inline src/main/resources/default-workflows/Machine_Learning/DLM_AnoDet_a_NS.xml]: ?

File: src/main/resources/default-workflows/Machine_Learning/DLM_AnoDet_n_NS.xml [hunk 4]
```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="DLM_AnoDet_n_NS" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2">
```
Comment (mbenguig) [inline src/main/resources/default-workflows/Machine_Learning/DLM_AnoDet_n_NS.xml]: ?

File: .gitignore (hunk 1 above)
Comment (sophiesongge) [reply .gitignore:11]: In Mac_OS, .DS_Store is a file that stores custom attributes of its containing folder, such as the position of icons or the choice of a background image. No need to commit them.

Comment (sophiesongge) [reply .gitignore:11]: So I add it to .gitignore file

File: src/main/resources/default-workflows/Machine_Learning/DLM_ClassImg_NS.xml [hunk 5]
```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="DLM_ClassImg_NS" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2">
```
Comment (mbenguig) [inline src/main/resources/default-workflows/Machine_Learning/DLM_ClassImg_NS.xml]: ?

File: src/main/resources/default-workflows/Machine_Learning/DLM_ObjDet_NS.xml [hunk 6]
```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="DLM_ObjDet_NS" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2">
```
Comment (mbenguig) [inline src/main/resources/default-workflows/Machine_Learning/DLM_ObjDet_NS.xml]: ?

File: src/main/resources/default-workflows/Machine_Learning/TensorFlow_Demo_Parallel_Prediction.xml [hunk 7]
```diff
@@ -0,0 +1,379 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="TensorFlow_Demo_Parallel_Prediction" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2">
```
Comment (mbenguig) [inline src/main/resources/default-workflows/Machine_Learning/TensorFlow_Demo_Parallel_Prediction.xml]: ?

File: src/main/resources/default-workflows/Machine_Learning/TensorFlow_Demo_Prediction.xml [hunk 8]
```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="TensorFlow_Demo_Prediction" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2">
```
Comment (mbenguig) [inline src/main/resources/default-workflows/Machine_Learning/TensorFlow_Demo_Prediction.xml]: ?

File: src/main/resources/default-workflows/Machine_Learning/TensorFlow_Demo_Training.xml [hunk 9]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="TensorFlow_Demo_Training" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2">
```
Comment (mbenguig) [inline src/main/resources/default-workflows/Machine_Learning/TensorFlow_Demo_Training.xml]: ?

File: src/main/resources/default-workflows/Machine_Learning/tensorflow_nodesource_delete.xml [hunk 10]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="tensorflow_nodesource_delete" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2">
```
Comment (mbenguig) [inline src/main/resources/default-workflows/Machine_Learning/tensorflow_nodesource_delete.xml]: ?

File: src/main/resources/default-workflows/Machine_Learning/tensorflow_nodesource_deploy.xml [hunk 11]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<job
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="urn:proactive:jobdescriptor:3.8"
+     xsi:schemaLocation="urn:proactive:jobdescriptor:3.8 http://www.activeeon.com/public_content/schemas/proactive/jobdescriptor/3.8/schedulerjob.xsd"
+    name="tensorflow_nodesource_deploy" 
+    priority="normal"
+    onTaskError="continueJobExecution"
+     maxNumberOfExecution="2">
```
Comment (mbenguig) [inline src/main/resources/default-workflows/Machine_Learning/tensorflow_nodesource_deploy.xml]: ?

File: src/main/resources/default-workflows/Machine_Learning/DLM_AnoDet_a_NS.xml (hunk 3 above)
Comment (sophiesongge) [reply src/main/resources/default-workflows/Machine_Learning/DLM_AnoDet_a_NS.xml]: Strange, no idea what is it. When I export it from Studio, it seems to be fine, but when uploaded in github, there are two more characters.

image

**Review by mbenguig — APPROVED** [review]

**Review by carolinepacheco — APPROVED** [review]

**Pull request merged**